io.github.RobertBeckebans.RBDoom3BFG.GL.appdata.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>io.github.RobertBeckebans.RBDoom3BFG.GL</id>
```

Comment:
this is incorrect, this should not have `.GL`

Reply:
If I remember it correctly, this Flatpak is intentionally using an older branch of RBDoom3BFG that only supports the OpenGL renderer.

Reply:
so the app name would be `RBDoom3BFG-GL` or something like that.